### ISSUE TITLE
Cumulus: BGP neighbor update-source not recognized

### ISSUE THREAD

**Issue by kaminek** [batfish/batfish#5162]
Hello I have a cumulus route reflector with the configuration below:
```
rr61
# This file describes the network interfaces
# This file describes the network interfaces available on your system
# and how to activate them. For more information, see interfaces(5).

# The loopback network interface
auto lo
iface lo inet loopback
    address 10.0.0.61/32

auto mgmt
iface mgmt
    vrf-table auto

auto swp1
iface swp1
    address 10.0.61.0/31

auto swp2
iface swp2
    address 10.0.61.64/31

# ports.conf --
# ports.conf --
#
#   configure port speed, aggregation, and subdivision.
#
#   The ports in Cumulus VX are not configurable from here.
#frr version

frr version
frr defaults datacenter
hostname rr61
username cumulus nopassword
!
service integrated-vtysh-config
!
log syslog informational
!
router bgp 64603
 bgp router-id 10.0.0.61
 neighbor peer-leaf peer-group
 neighbor peer-leaf remote-as 64601
 neighbor peer-leaf bfd 3 100 100
 neighbor peer-leaf-overlay peer-group
 neighbor peer-leaf-overlay remote-as 64603
 neighbor peer-leaf-overlay bfd 3 100 100
 neighbor peer-leaf-overlay update-source lo
 neighbor 10.0.61.1 peer-group peer-leaf
 neighbor 10.0.61.1 description leaf1
 neighbor 10.10.0.1 peer-group peer-leaf-overlay
 neighbor 10.10.0.1 description leaf1

 address-family ipv4 unicast
  redistribute connected route-map redistribute-only-lo
  neighbor peer-leaf activate
 exit-address-family
 !
 address-family l2vpn evpn
  neighbor peer-leaf-overlay activate
 exit-address-family
!
route-map redistribute-only-lo permit 10
 match interface lo
!
line vty

```

the BGP sessions with peer-leaf-overlay are not established with `LOCAL_IP_UNKNOWN_STATICALLY` because the source IP is not set because of:
```
Parse warning | This syntax is unrecognized | neighbor peer-leaf-overlay interface lo | [sbn_property sbn_interface sbn_name sb_neighbor s_bgp statement cumulus_frr_configuration]
```
Even the ip address is not considered correct
```
neighbor peer-leaf-overlay update-source 10.0.0.61
```

**Comment by ratulm** [comment]
Fixed by #5184

**Comment by yifeiyuan** [comment]
fixed by #5184
